### PR TITLE
Delete json-schemas and pull hosted ones.

### PR DESCRIPTION
**Pull request by mistermoe**
Fetches JSON schemas from https://tbdex.dev which pulls directly from [tbdex github repo](https://github.com/TBD54566975/tbdex/tree/main/json-schemas). 

> [!NOTE] 
> tests will fail until https://github.com/TBD54566975/tbdex/pull/170 is merged!

### CODE REVIEW

**Comment by mistermoe** [comment]
related: https://github.com/TBD54566975/tbdex/pull/170

File: .github/workflows/integrity-check.yml [hunk 1]
```diff
@@ -64,32 +64,32 @@ jobs:
...
-
-      - name: Install dependencies
-        run: pnpm install
-
-      - name: Install Playwright Browsers
-        run: npx playwright install --with-deps
-
-      - name: Run tests for all packages
-        run: pnpm test:browser
+  # test-with-browsers:
```
Comment (KendallWeihe) [inline .github/workflows/integrity-check.yml:67]: commented out b/c why?

**Review by KendallWeihe — APPROVED** [review]

File: .github/workflows/integrity-check.yml (hunk 1 above)
Comment (mistermoe) [reply .github/workflows/integrity-check.yml:67]: runs indefinitely for days without exiting

**Comment by mistermoe** [comment]
@leordev any idea why the `tbdocs-reporter` action might be failing?

**Comment by leordev** [comment]
> @leordev any idea why the `tbdocs-reporter` action might be failing?

Yes, it's because the tbdocs report inputs params had a break change and we need to merge #42 to fix it! You can go ahead and merge it and rebase this one against main, or just ignore it.

**Review by KendallWeihe — APPROVED** [review]
> > @leordev any idea why the `tbdocs-reporter` action might be failing?
> 
> Yes, it's because the tbdocs action inputs params had a breaking change and we need to merge #42 to fix it! You can go ahead and merge it and rebase this one against main, or just ignore it.

Merged the other open PR's and rebased this one

@mistermoe this can be merged, I'll let you do it

**Review by jiyoontbd — APPROVED** [review]

**Comment by leordev** [comment]
> > @leordev any idea why the `tbdocs-reporter` action might be failing?
> 
> Yes, it's because the tbdocs action inputs params had a breaking change and we need to merge #42 to fix it! You can go ahead and merge it and rebase this one against main, or just ignore it.

I saw the error persists, it's because of a bug on the git diff function. I will be working on it from the plane. But I dont want to block you, feel free to ignore for now and merge.

**Comment by KendallWeihe** [comment]
> so does this mean we don't have to do submodule tings in tbdex-kt?

We would have to implement the same logic (to download schemas from host) over in `tbdex-kt`, but yeah 🎉

**Pull request merged**